configuration/device-tree.md

```diff
@@ -405,56 +479,58 @@ Here are some examples of different types of properties, with parameters to modi
 		u32_1 =       <&test>,"u32s:4";
 		u64_0 =       <&test>,"u64s#0";
 		u64_1 =       <&test>,"u64s#8";
-		bool1 =       <&test>,"bool1?";
+		bool1 =       <&test>,"bool1!";
 		bool2 =       <&test>,"bool2?";
-		only1 =       <0>,"+1-2";
-		only2 =       <0>,"-1+2";
-		toggle1 =     <0>,"=1";
+		entofr =      <&test>,"english",
+		              <&test>,"french{hello=bonjour,goodbye='au revoir',weekend}";
+		pi_mac =      <&test>,"mac[{1=b8273bfedcba,2=b8273b987654}";
+		spibus =      <&test>,"spi:0[0=",<&spi0>,"1=",<&spi1>,"2=",<&spi2>;
+
 		toggle2 =     <0>,"=2";
 		not1 =        <0>,"!1";
-		not2 =        <0>,"!2";
+
+		only1 =       <0>,"+1-2";
+		only2 =       <0>,"-1+2";
+		enable1 =     <0>,"=1";
+		disable2 =    <0>,"!2";
 	};
 };
 ```
 
-<a name="part2.2.6"></a>
-#### 2.2.6: Parameters with multiple targets
-
-There are some situations where it is convenient to be able to set the same value in multiple locations within the Device Tree. Rather than the ungainly approach of creating multiple parameters, it is possible to add multiple targets to a single parameter by concatenating them, like this:
-
-```
-    __overrides__ {
-        gpiopin = <&w1>,"gpios:4",
-                  <&w1_pins>,"brcm,pins:0";
-        ...
-    };
-```
+For further examples, there is a large collection of overlay source files hosted in the Raspberry Pi/Linux GitHub repository [here](https://github.com/raspberrypi/linux/tree/master/arch/arm/boot/dts/overlays).
```

Comment:
Probably remove the slash to make it just "Raspberry Pi Linux GitHub repository" ?

Reply:
Will do.